client/src/components/sidebar/sidebar-item-content.tsx

```diff
@@ -56,10 +56,9 @@ const SidebarItemContent = forwardRef<HTMLDivElement, Props>(
                 {(isSub || active) && (
                     <div
                         className={cn(
-                            'absolute left-[-11px] size-1.5 rounded-full bg-secondary duration-300 ease-in-out',
+                            'absolute left-[-11px] size-1.5 rounded-full bg-secondary delay-100 duration-300 ease-out group-hover/navself:bg-primary',
                             active && 'bg-primary',
```

Comment:
Suggestion
When using boolean states, it is better to use proper naming. Example: Instead of `active`, use `isActive`.